src/herder/HerderSCPDriver.cpp

```diff
@@ -706,6 +720,16 @@ HerderSCPDriver::combineCandidates(uint64_t slotIndex,
                 {
                     highestTxSet = cTxSet;
                     highest = sv.txSetHash;
+                    bestTxSetCloseTime = sv.closeTime;
+                    if (newCloseTimeBehavior)
+                    {
+                        if (sv.ext.v() != STELLAR_VALUE_SIGNED)
```

Comment:
> Isn't this already checked by validateValueHelper?

Yeah, I just meant it as an assertion that that method had done what combineCandidates() expects.  If this kind of assertion is more confusing than clarifying, I will try to start avoiding them.

> Also, you should be able to simply use the candidate comp = sv like it's specified in the proposal instead of manually crafting a new value like this

We're not taking `sv.upgrades`, though, are we?  That's still a combination of all candidates' upgrades.  Are you saying it would be clearer to do `comp = sv` and then overwrite the `upgrades`?  I thought that might have been misleading (back at the location of the `comp = sv`).